control-plane/plugin/src/resources/node.rs

```diff
@@ -105,18 +111,98 @@ impl List for Nodes {
     }
 }
 
+/// The NodeDisplayLabels structure is responsible for controlling the display formatting of Node
+/// objects. `#[serde(flatten)]` and `#[serde(skip)]` attributes are used to ensure that when the
+/// object is serialised, only the `inner` object is represented.
+#[derive(Serialize, Debug)]
+pub struct NodeDisplayLabels {
+    #[serde(flatten)]
+    pub inner: Vec<openapi::models::Node>,
+    #[serde(skip)]
+    show_labels: bool,
```

Comment:
Is this needed?
Shouldn't show_labels be always true when we use this type?

Reply:
no, its true when `--show-labels` is passed